### PR TITLE
minor improvements in GeoMechanicsApplication

### PR DESCRIPTION
**Pull request by Vahid-Galavi**
**Description**
Some minor improvments suggested in the previous PR.

### CODE REVIEW

File: applications/GeoMechanicsApplication/custom_constitutive/small_strain_udsm_2D_plane_strain_law.cpp [hunk 1]
```diff
@@ -206,9 +203,10 @@ Vector& SmallStrainUDSM2DPlaneStrainLaw::
 }
 
 //----------------------------------------------------------------------------------------
-void SmallStrainUDSM2DPlaneStrainLaw::SetValue( const Variable<Vector>& rThisVariable,
-                                                const Vector& rValue,
-                                                const ProcessInfo& rCurrentProcessInfo )
+void SmallStrainUDSM2DPlaneStrainLaw::
+   SetValue( const Variable<Vector>& rThisVariable,
+             const Vector& rValue,
+             const ProcessInfo& rCurrentProcessInfo )
```
Comment (rubenzorrilla) [inline applications/GeoMechanicsApplication/custom_constitutive/small_strain_udsm_2D_plane_strain_law.cpp:209]:
```suggestion
void SmallStrainUDSM2DPlaneStrainLaw::SetValue(
    const Variable<Vector>& rThisVariable,
    const Vector& rValue,
    const ProcessInfo& rCurrentProcessInfo )
```
We normally use this style convention for methods with more than one argument.

File: applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_2D_plane_strain_law.cpp [hunk 2]
```diff
@@ -78,54 +78,56 @@ SmallStrainUMAT2DPlaneStrainLaw::~SmallStrainUMAT2DPlaneStrainLaw() {}
...
+   }
 
    KRATOS_CATCH("")
 }
 
-void SmallStrainUMAT2DPlaneStrainLaw::CopyConstitutiveMatrix( ConstitutiveLaw::Parameters &rValues,
-                                                              Matrix& rConstitutiveMatrix )
+void SmallStrainUMAT2DPlaneStrainLaw::
+   CopyConstitutiveMatrix( ConstitutiveLaw::Parameters &rValues,
+                           Matrix& rConstitutiveMatrix )
```
Comment (rubenzorrilla) [inline applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_2D_plane_strain_law.cpp:130]:
```suggestion
void SmallStrainUMAT2DPlaneStrainLaw::CopyConstitutiveMatrix(
    ConstitutiveLaw::Parameters &rValues,
    Matrix& rConstitutiveMatrix)
```

File: applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_2D_plane_strain_law.cpp [hunk 3]
```diff
@@ -205,9 +203,10 @@ Vector& SmallStrainUMAT2DPlaneStrainLaw::
 }
 
 //----------------------------------------------------------------------------------------
-void SmallStrainUMAT2DPlaneStrainLaw::SetValue( const Variable<Vector>& rThisVariable,
-                                                const Vector& rValue,
-                                                const ProcessInfo& rCurrentProcessInfo )
+void SmallStrainUMAT2DPlaneStrainLaw::
+   SetValue( const Variable<Vector>& rThisVariable,
+             const Vector& rValue,
+             const ProcessInfo& rCurrentProcessInfo )
```
Comment (rubenzorrilla) [inline applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_2D_plane_strain_law.cpp:209]:
```suggestion
void SmallStrainUMAT2DPlaneStrainLaw::SetValue(
    const Variable<Vector>& rThisVariable,
    const Vector& rValue,
    const ProcessInfo& rCurrentProcessInfo)
```

**Review by rubenzorrilla — APPROVED** [review]
Just a few minor style guide suggestions. Aside of this changes are OK to me.

**Comment by Vahid-Galavi** [comment]
> Just a few minor style guide suggestions. Aside of this changes are OK to me.

Thanks @rubenzorrilla! Is there a programming style guid for Kratos?

**Comment by rubenzorrilla** [comment]
> > Just a few minor style guide suggestions. Aside of this changes are OK to me.
> 
> Thanks @rubenzorrilla! Is there a programming style guid for Kratos?

[Here](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) it is. :wink:

**Comment by Vahid-Galavi** [comment]
> > > Just a few minor style guide suggestions. Aside of this changes are OK to me.
> > 
> > 
> > Thanks @rubenzorrilla! Is there a programming style guid for Kratos?
> 
> [Here](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide) it is.

Thanks @rubenzorrilla!

**Pull request merged**